ArduCopter/mode_autorotate.cpp

```diff
@@ -66,6 +61,10 @@ bool ModeAutorotate::init(bool ignore_checks)
 
     // The decay rate to reduce the head speed from the current to the target
     _hs_decay = ((_initial_rpm/g2.arot.get_hs_set_point()) - HEAD_SPEED_TARGET_RATIO) / AUTOROTATE_ENTRY_TIME;
+	
+    //set motors to idle
+    copter.ap.motor_interlock_switch = false;
```

Comment:
This looks like a big no no to me.  We should not be setting the interlock from the code.  Why do we need to do this anyway?

Reply:
this is specific for ICE in case of "real" emergency. It is safer to have the ECU set to throttle-off position in case of in-flight shutdown/flameout.